RTN-095.tex

```diff
@@ -1,4 +1,6 @@
-\documentclass[twocolappendix, trackchanges]{aastex7}
+%\documentclass[twocolappendix, trackchanges]{aastex7}
+\documentclass[twocolumn,linenumbers,trackchanges]{aastex7}
```

Comment:
```suggestion
\documentclass[twocolumn,twocolappendix,linenumbers,trackchanges]{aastex7}
```

Reply:
I prefer to have twocolappendix also  - the sed removes two col but leave two col appendix - it may make no difference but ultimately the glossary ends up twocolumn